components/google_calendar/actions/get-acl/get-acl.js

```diff
@@ -0,0 +1,34 @@
+const googleCalendar = require("../../google_calendar.app");
+
+module.exports = {
+  key: "google_calendar_get_acl",
+  name: "Retreive Access Control Rule",
+  description: "Retreive Access Control Rule Metadata of a google calendar.",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    googleCalendar,
+    calendarId: {
+      propDefinition: [
+        googleCalendar,
+        "calendarId",
+      ],
+    },
+    ruleId: {
+      propDefinition: [
+        googleCalendar,
+        "ruleId",
+        (c) => ({
+          calendarId: c.calendarId,
+        }),
+      ],
+    },
+  },
+  async run() {
+    const calendar = this.googleCalendar.calendar();
+    return (await calendar.acl.get({
+      calendarId: this.calendarId,
+      ruleId: this.ruleId,
+    })).data;
```

Comment:
I would recommend creating a new function named `getAcl` in `google_calendar.app.js` like this 
```javascript
    async getAcl(params) {
      const calendar = this.calendar();
      const { data } = await calendar.acl.get(params);
      return data;
    },
```
following the same pattern as the other functions. Then you can call the function like this
```javascript
    return this.googleCalendar.getAcl({
      calendarId: this.calendarId,
      ruleId: this.ruleId,
    });
```
And the same pattern applies to `get-calendar.js, get-event.js, list-acl.js, quick-add-event.js, update-acl.js`